data/json/items/tool/electronics.json

```diff
@@ -50,7 +50,7 @@
   {
     "id": "camera_pro",
     "type": "TOOL",
-    "name": "camera pro",
+    "name": { "str": "camera pro" },
```

Comment:
I am not sure about "camera pros" being valid.

Reply:
I changed it thanks